engine/src/lib.rs

```diff
@@ -142,6 +144,20 @@ mod contract {
         io.return_output(state.owner_id.as_bytes());
     }
 
+    /// Set owner account id for this contract.
+    #[no_mangle]
+    pub extern "C" fn set_owner() {
```

Comment:
This is a new public method, so it requires changes to the Borealis Engine code as well. Please add a new transaction kind [here](https://github.com/aurora-is-near/aurora-engine/blob/master/engine-standalone-storage/src/sync/types.rs#L76) and follow the compilation errors that arise (unexhaustive match) to finish the implementation.

Reply:
TransactionKind is now added, borsh inputs are added and tested with workspace. https://github.com/hskang9/aurora-silo-tests/blob/05f05505d9c362c3b2648bfa7647474689014271/src/test.rs#L154